custom/stdweb/src/lib.rs

```diff
@@ -6,26 +6,28 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Implementation for WASM via stdweb
-extern crate std;
+//! `getrandom` implementation for WASM via stdweb
+#![cfg(all(target_arch = "wasm32", target_os = "unknown"))]
```

Comment:
I think `compilation_error!` would be a bit more clear, something liek this::
```rust
#[cfg(not(all(target_arch = "wasm32", target_os = "unknown")))]
compilation_error!("This crate can be used only on `wasm32-unknown-unknown` target");
```

Reply:
Done